DeNovo.pm

```diff
@@ -118,19 +123,27 @@ sub new {
     $self->{report_dir} = "$cwd_dir";
   }
 
+  if (defined $param_hash->{full_report}) {
+    $self->{full_report} = 1;
+  }
+
   # force some config params
   $self->{config}->{individual_zyg} = ['all'];
 
   # Report files for each family
   my $family_list = $self->{family_list};
   foreach my $family (keys %{$family_list}) {
     $self->{report_de_novo}->{$family} = $family . "_variants_de_novo.txt";
-    $self->{report_both_parents}->{$family} = $family . "_variants_both_parents.txt";
-    $self->{report_all}->{$family} = $family . "_variants_child_and_both_parents.txt";
     # Write header
     write_header($self->{report_dir}.'/'.$self->{report_de_novo}->{$family}, 1);
-    write_header($self->{report_dir}.'/'.$self->{report_both_parents}->{$family});
-    write_header($self->{report_dir}.'/'.$self->{report_all}->{$family});
+
+    if($self->{full_report}) {
+      $self->{report_both_parents}->{$family} = $family . "_variants_both_parents.txt";
```

Comment:
To avoid deleting files by mistake the plugin could die if there are files in the directory. Then the user chooses if they want to delete the files or point to another directory.